source/include/drivers/cs_PWM.h

```diff
@@ -10,6 +10,8 @@
 
 #include "ble/cs_Nordic.h"
 #include "cfg/cs_Config.h"
+#include <components/libraries/pwm/app_pwm.h>
```

Comment:
Since when do we use this header?

Reply:
replaced with headers from `modules/nrfx/hal/nrf_*`